frontend/src/ramp.pug

```diff
@@ -23,7 +23,8 @@
       onclick="rampClick(event);",
       v-for="(slice, j) in user.commits",
       v-if="slice.insertions > 0",
-      v-bind:title="`${tframe === 'day' ? '[' + slice.date + '] Daily' : '[' + slice.date + ' till ' + slice.endDate + '] Weekly'} contribution: ${slice.insertions} lines`",
+      v-bind:title="`${tframe === 'day' ? '[' + slice.date + '] Daily'\
```

Comment:
Perhaps adding another new line before `?` would be clearer to show the turnery operator